Documentation/Functions/Typolink.rst

```diff
@@ -183,17 +183,30 @@ addQueryString
    not check for any duplicate parameters! This is not a problem: Only
    the last parameter of the same name will be applied.
 
-   .method
-      If set to GET or POST, then the parsed query arguments
-      (GET or POST data) will be used. This setting is useful, if you use
-      URL processing extensions like Real URL, which translate part of the
-      path into query arguments.
+   .. attention::
+
+      .. versionchanged:: 10.0
+
+         With version 10.0 a breaking change was introduced. Setting
+         :typoscript:`addQueryString.method` to any value but :typoscript:`GET` will
+         trigger a `E_USER_WARNING`. Any POST-parameters will be ignored. Formerly this option
+         was set to :typoscript:`POST` or some combination in order to forward
+         POST-parameters as GET parameters.
+
+         In terms of correctly using HTTP verbs it’s bad practise in
+         general to treat GET and POST equally, besides that documentation
+         already mentioned potential side-effects like accidentally exposing
+         sensitive data submitted via POST to proxies or log files.
```

Comment:
```suggestion
         In terms of correctly using HTTP verbs, it is bad practice in
         general to treat GET and POST equally. In addition, there can be
         potential side-effects like accidentally exposing sensitive data 
         submitted via POST to proxies or log files.
```
I wasn't sure about including the documentation if it's unclear where that was. Perhaps you could link to where it already mentioned it.